CMakeLists.txt

```diff
@@ -71,10 +71,10 @@ option(FLATBUFFERS_ENABLE_PCH
         Only work if CMake supports 'target_precompile_headers'. \"
         This can speed up compilation time."
        OFF)
-option(FLATBUFFERS_SKIP_MONSTER_EXTRA 
+option(FLATBUFFERS_SKIP_MONSTER_EXTRA
```

Comment:
Note, I would prefer these formatting changes in a separate PR, but its ok for this one.